### PR TITLE
Gangams/updates related 1p mode

### PR DESCRIPTION
**Pull request by ganga1980**
This PR has following changes related to 1p

1.  config xml with 1 sec upload frequency
2.  Grafana dashboard 
3.  HPA updates
4.  Geneva chart updates related MDSD environment variables for high log scale
5.  test workloads

### CODE REVIEW

**Review by wanlonghenry — APPROVED** [review]

File: Documentation/Internal/Grafana/AzureMonitorContainers-GenevaLogs_Multitenancy-Platform.json [hunk 1]
```diff
@@ -0,0 +1,1134 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": {
+          "type": "grafana",
+          "uid": "-- Grafana --"
```
Comment (wanlonghenry) [inline Documentation/Internal/Grafana/AzureMonitorContainers-GenevaLogs_Multitenancy-Platform.json:8]: please double check that uid is this format

File: Documentation/Internal/Grafana/ama-metrics-prometheus-config.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+kind: ConfigMap
+apiVersion: v1
+data:
+  prometheus-config: |-
+    scrape_configs:
+    - job_name: ama-logs-daemonset-linux
+      kubernetes_sd_configs:
+        - role: pod
+      relabel_configs:
+        - source_labels: [__meta_kubernetes_pod_controller_kind]
```
Comment (wanlonghenry) [inline Documentation/Internal/Grafana/ama-metrics-prometheus-config.yaml:10]: do you know why some labels is in this format __<lable>__ vs __<lable>

File: charts/azuremonitor-containers-geneva/templates/deployment.yaml [hunk 3]
```diff
@@ -61,12 +61,27 @@ spec:
...
-         - name: MONITORING_MAX_EVENT_RATE
-           value: "100000" # default MDSD EPS is 20K which is not enough for large scale
          - name: FBIT_INPUT_FORWARD_BUFFER_CHUNK_SIZE
            value: "15m"
          - name: FBIT_INPUT_FORWARD_BUFFER_CHUNK_MAX_SIZE
            value: "30m"
+         # MDSD high scale config
+         - name: MONITORING_MAX_EVENT_RATE
+           value: "100000" # default MDSD EPS is 20K which is not enough for large scale
+         - name: MDSD_COMPRESSION_ALGORITHM
```
Comment (jatakiajanvi12) [inline charts/azuremonitor-containers-geneva/templates/deployment.yaml:71]: Can we add the meaning of all these vars in our wiki or something?

**Review by jatakiajanvi12 — COMMENTED** [review]
Can we add a better name of the PR since we are going to use this directly in release notes as last time?

File: Documentation/Internal/Grafana/AzureMonitorContainers-GenevaLogs_Multitenancy-Platform.json (hunk 1 above)
Comment (ganga1980) [reply Documentation/Internal/Grafana/AzureMonitorContainers-GenevaLogs_Multitenancy-Platform.json:8]: > please double check that uid is this format

This is correct. This is generated by Grafana tool.

File: Documentation/Internal/Grafana/ama-metrics-prometheus-config.yaml (hunk 2 above)
Comment (ganga1980) [reply Documentation/Internal/Grafana/ama-metrics-prometheus-config.yaml:10]: > do you know why some labels is in this format vs __

These are standard meta labels - https://prometheus.io/docs/prometheus/latest/configuration/configuration/#pod

**Comment by ganga1980** [comment]
> Can we add a better name of the PR since we are going to use this directly in release notes as last time?

> please note that, none of these changes related to agent and changes are already in the description already.

File: charts/azuremonitor-containers-geneva/templates/deployment.yaml (hunk 3 above)
Comment (ganga1980) [reply charts/azuremonitor-containers-geneva/templates/deployment.yaml:71]: > Can we add the meaning of all these vars in our wiki or something?

These are all MDSD specific. let me check with MDSD team and see if they have doc for this.

**Pull request merged**